starter-code/css/stayle.css

```diff
@@ -0,0 +1,98 @@
+header {
+    border-top: 8px solid black;
+    margin: 20px 10px 10px 10px;

```

Comment:
you can also write it as: 
margin: 20px 10px 10px; 
and both the left and right margins will take 10px margin since you didn't specify it will automatically take the value of the opposite side. And by specify I mean you didn't write:
margin: 20px 10px 10px 0px; 
In the above case it will make the left margin 0px;